sycl/doc/GlobalObjectsInRuntime.md

```diff
@@ -88,7 +88,10 @@ constructor and destructor.
 
 ## Plugins
 
-TBD
+Plugin lifetime is managed by utilizing initializePlugins() and piTearDown(). GlobalHandler::shutdown() will tear down all registered globals at the end of the Sycl program execution. It will invoke piTearDown() and unload() for each plugin. unload() is going to eventually invoke OS-specific system call to remove the plugin dynamic library from memory. piTearDown() is going to perform any necessary tear-down process at the plugin PI level. Currently, it is doing global memory deallocation to avoid memory leaks. But, in the future, it can include any other jobs that need to be done before we unload the plugin from memory. This enables on-deman plugin lifetime management. Sycl RT can control the beginning and the end of the plugin. 
```

Comment:
The official spelling is SYCL, all capitals. Also, please keep lines ~80 char long, as the rest of this document. Makes it easier to read from text editors.

```suggestion
Plugin lifetime is managed by utilizing initializePlugins() and piTearDown(). GlobalHandler::shutdown() will tear down all registered globals at the end of the SYCL program execution. It will invoke piTearDown() and unload() for each plugin. unload() is going to eventually invoke OS-specific system call to remove the plugin dynamic library from memory. piTearDown() is going to perform any necessary tear-down process at the plugin PI level. Currently, it is doing global memory deallocation to avoid memory leaks. But, in the future, it can include any other jobs that need to be done before we unload the plugin from memory. This enables on-deman plugin lifetime management. SYCL RT can control the beginning and the end of the plugin. 
```